File: 백준/Bronze/13496. The Merchant of Venice/The Merchant of Venice.c
#include <stdio.h>
int main(void) {
    int k, n, s, d, di, vi, total;
    scanf("%d", &k);
    for (int i = 1; i < k + 1; i++) {
        scanf("%d %d %d", &n, &s, &d);
        total = 0;
        for (int j = 0; j < n; j++) {
            scanf("%d %d", &di, &vi);
            if (di / s + (di % s == 0 ? 0 : 1) <= d)
                total += vi;
        }
        printf("Data Set %d:\n%d\n\n", i, total);
    }
    return 0;
}

File: 백준/Bronze/13496. The Merchant of Venice/The Merchant of Venice.py
import sys
input=lambda:sys.stdin.readline().rstrip()
k=int(input())
for i in range(1,k+1):
    total=0
    n,s,d=map(int,input().split())
    for _ in range(n):
        di,vi=map(int,input().split())
        if di//s+(0 if di%s==0 else 1)<=d:
            total+=vi
    print(f"Data Set {i}:\n{total}\n")